app/logic/Mail/JSON/JSONEMail.ts

```diff
@@ -203,7 +203,7 @@ export class JSONEMail {
       a.mimeType = sanitize.nonemptystring(json.mimeType, "application/octet-stream");
       a.contentID = sanitize.nonemptystring(json.contentID, "" + fallbackID);
       a.filename = sanitize.nonemptystring(json.filename, "attachment-" + fallbackID + "." + fileExtensionForMIMEType(a.mimeType));
-      let filepathLocal = sanitize.string(json.filepathLocal, null)
+      let filepathLocal = sanitize.string(json.filepath, null)
```

Comment:
Why not `Local` here?

Reply:
I just used the property name in the `save()` method. But I see you fixed it thanks!